### DISCUSSION TITLE
file permissions fro files downloaded from onedrive

### DISCUSSION THREAD

**Discussion by fbuescher** [abraunegg/onedrive#1326] [Q&A] [answered]
Hi,

when onedrive downloads new files from cloud, it creates the files with permissions 600 and directories with permission 700.
is there any way to make onedrive create files with 664 and directories with 775? 
I tried 

`umask 0002`
`/usr/local/bin/onedrive --monitor`

but onedrive ignores this setting.

**Comment by fbuescher** [comment, marked as answer]
Please ignore, i just found the configs


sync_dir_permissions
sync_file_permissions


thanks,
Friedhelm

**Comment by abraunegg** [comment]
@fbuescher 
Correct - this is also documented here: https://github.com/abraunegg/onedrive/blob/master/docs/USAGE.md#sync_dir-directory-and-file-permissions
